contentgrid-spring-audit-logging/src/main/java/com/contentgrid/spring/audit/handler/messaging/AuditEventToCloudEventMessageConverter.java

```diff
@@ -0,0 +1,94 @@
+package com.contentgrid.spring.audit.handler.messaging;
+
+import com.contentgrid.spring.audit.event.AbstractAuditEvent;
+import com.contentgrid.spring.audit.event.AbstractEntityRelationAuditEvent;
+import com.contentgrid.spring.audit.event.BasicAuditEvent;
+import com.contentgrid.spring.audit.event.EntityContentAuditEvent;
+import com.contentgrid.spring.audit.event.EntityItemAuditEvent;
+import com.contentgrid.spring.audit.event.EntityItemAuditEvent.Operation;
+import com.contentgrid.spring.audit.event.EntitySearchAuditEvent;
+import io.cloudevents.core.builder.CloudEventBuilder;
+import io.cloudevents.core.data.PojoCloudEventData;
+import java.net.URI;
+import java.util.Locale;
+import java.util.Set;
+import java.util.UUID;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.MessageHeaders;
+import org.springframework.messaging.converter.MessageConverter;
+
+@RequiredArgsConstructor
+public class AuditEventToCloudEventMessageConverter implements MessageConverter {
+
+    @NonNull
+    private final MessageConverter delegate;
+
+    @NonNull
+    private final PojoCloudEventData.ToBytes<AbstractAuditEvent> mapper;
+
+    @NonNull
+    private final URI source;
+
+    private static final Set<AuditEventToCloudEventMapper> TYPE_MAPPING = Set.of(
+            AuditEventToCloudEventMapper.build(BasicAuditEvent.class)
+                    .cloudEventType("cloud.contentgrid.audit.basic")
```

Comment:
It's `cloud.contentgrid` prefiexed because it is runtime ?

Reply:
Yes, and the cloud event type should be in reverse-DNS format according to the cloud-events spec